hugo/layouts/community/single.html

```diff
@@ -23,6 +23,7 @@ <h2 class="title-line line-yellow">Our Community Channels</h2>
                         <a href="https://forum.sourced.tech" class="text-dark">
                             <h3 class="title-line line-yellow">source{d} Forum</h3>
                             <p>This is the forum where users and community members discuss anything source{d} related. source{d} is the data platform for the software development life cycle. Save time and modernize your codebase by gaining visibility into your software portfolio, engineering team and processes.</p>
+                            <a class="btn btn-highlight col-md-6 mt-3" href="https://forum.sourced.tech">Forum</a>
```

Comment:
There's a link inside of the link, what is not HTML well-formed.
Now that we have clear CTA, I'd remove the outer link,